internal/slice/slice.go

```diff
@@ -9,3 +9,13 @@ func Contains[E comparable](s []E, v E) bool {
 	}
 	return false
 }
+
+// ContainsAny reports whether v is present in s
+func ContainsAny(s []any, v any) bool {
```

Comment:
nit: should we name package `slice` to package `slices` so that we can have consistent naming as https://pkg.go.dev/golang.org/x/exp/slices?